README.md

```diff
@@ -1,60 +1,77 @@
 <div align="center">
-<h1>⚛️ create-sumup-react-app ⚛️</h1>
+<h1>⚛️ @sumup/cra-template ⚛️</h1>
 
-The easiest way to get started writing a React app with SumUp's standard frontend toolchain.
+A
+[create-react-app template](https://create-react-app.dev/docs/custom-templates)
+for JavaScript React apps with SumUp's frontend stack.
 
 </div>
 
 ## Table of Contents <!-- omit in toc -->
 
-- [💻 Prerequisites](#%f0%9f%92%bb-prerequisites)
-- [✨ Setting up a new SumUp React project](#%e2%9c%a8-setting-up-a-new-sumup-react-project)
-- [🛠 Development workflow](#%f0%9f%9b%a0-development-workflow)
+- [💻 Prerequisites](#-prerequisites)
+- [✨ Setting up a new JavaScript SumUp React project](#-setting-up-a-new-javascript-sumup-react-project)
+- [🛠 Development workflow](#-development-workflow)
   - [Firing up the tools](#firing-up-the-tools)
   - [Adding a new component](#adding-a-new-component)
   - [Linting your code](#linting-your-code)
   - [Writing tests](#writing-tests)
-- [📖 Useful resources](#%f0%9f%93%96-useful-resources)
-- [💁‍♀ Tips](#%f0%9f%92%81%e2%80%8d%e2%99%80-tips)
+- [📖 Useful resources](#-useful-resources)
+- [💁‍♀ Tips](#-tips)
   - [Setting up Foundry's ESLint in your editor](#setting-up-foundrys-eslint-in-your-editor)
     - [Visual Studio Code](#visual-studio-code)
-- [⚠️ Troubleshooting](#%e2%9a%a0%ef%b8%8f-troubleshooting)
+- [⚠️ Troubleshooting](#️-troubleshooting)
   - [Debug mode](#debug-mode)
-  - [Engine node is incompatible](#engine-node-is-incompatible)
-  - [Error watching file for changes](#error-watching-file-for-changes)
+  - [Broken build due to ESLint version](#broken-build-due-to-eslint-version)
 - [Code of conduct (CoC)](#code-of-conduct-coc)
   - [Maintainers](#maintainers)
 - [About SumUp](#about-sumup)
 
 ## 💻 Prerequisites
 
-- You should have **[Node.js](https://nodejs.org/)** installed at a version equal or above **`v9.10.0`**. If you cannot do that, see the [Troubleshooting](#troubleshooting) section for ways around this requirement.
-- At the moment this project reqires **[`yarn`](https://yarnpkg.com/)** to be installed on your system. Yarn is a package manager for JavaScript. You can read how to install the Yarn CLI in [their documentation](https://yarnpkg.com/en/docs/install).
-- You will need **`npx`** installed. You can run `yarn global add npx` or `npm install -g npx` to make that happen.
+- You should have **[Node.js](https://nodejs.org/)** installed at a version
+  equal or above **`v14.0.0`**. If you cannot do that, see the
```

Comment:
What's your motivation for introducing these forced linewraps? 

To be honest, I find that they make the text harder to read and code review becomes more difficult because text shifts across multiple lines when modified 😕

Reply:
Ah, sorry. So, actually I thought I had put this in the prettier config but it seems I did not.

> To be honest, I find that they make the text harder to read

In which context? Isn't it much easier to read prose at <80 character line width? You cannot always count on a view that you see raw markdown in to have soft-wrapping configured. And with prettier configured for Markdown reformatting the code is just a save or pre-commit hook away.

> code review becomes more difficult because text shifts across multiple lines when modified

True, a change causes shifts. At the same time an endless line for an entire paragraph is also somewhat difficult to read.

This is not a hard requirement for me. I'd be very curious to see a demo of the issues you describe, just to be able to understand the problem better.

Reply:
> In which context? Isn't it much easier to read prose at <80 character line width?

Generally, yes. IMO hard-coding this preference is not the way to go though. A simple scenario where this breaks is when my editor window is narrower than 80 chars. If I don't have softwrap enabled, the content overflows anyway and I need to scroll again. If I do have softwrap enabled, then I get orphans (single words on a line) because of the forced line breaks.

This is simple enough for each user to configure themselves. When the wrapping is done by the editor at "view time", it can intelligently reflow the text for the view context. When the wrapping is hard-coded at "write time", we loose this adaptability.

Reply:
I removed the hard wraps and the top-level prettier config override.